draft-ietf-quic-ack-frequency.md

```diff
@@ -479,8 +478,14 @@ new ACK_FREQUENCY frame immediately upon confirmation of connection migration.
 
 
 # Security Considerations
+
 TBD.
 
+QUIC ACK_FREQUENCY frames are protected by the normal quic security mechanisms {{QUIC-TRANSPORT}}). 
+Potentially an ACK_FREQUENCY frame could allow a DOS attack on the return path of a standards-compliant 
+receiver by a malicious/poorly configured server, however the sender has control of many aspects of 
+the receiver behaviour and this as a threat from a well-configured sender. 
+
```

Comment:
hmm, the first sentence can be rewritten as:

Improperly configured  or malicious senders could cause standards-compliant receivers to acknowledge more frequently than their available resources permit, thus causing a Denial Of Service (DOS) attack.  

I'm not sure about the second sentence, what was the gist of it? Senders have control of many aspects of the receiver behaviour anyway and so this is not an issue for well-configured servers?